default.xml

```diff
@@ -6,13 +6,17 @@
 
   <default remote="github" revision="master" sync-j="4"  />
 
+  <!-- Istio global build -->
+  <project path="build" name="istio/manifest"/>
```

Comment:
I was planning to check out test-infra repository as "/build" - and add the global
makefile and workspace to test-infra, with symlinks.

It is fine to also checkout the manifests - but maybe under /build/manifest ?  
I don't think the manifest needs a prow, it is typically narrowly used for defining the 
source layout ( with different orgs using different manifests to build different product
layouts)

Reply:
I don't think using test-infra is a good idea here, as we might have different build according to branches. test-infra should not have branches. I added a prow folder to eventually do presubmit (which I had and deleted), but I need to have the green-build job in there.

let's talk Monday in person, you may be thinking of something I am not seeing.